scripts/start

```diff
@@ -10,6 +10,8 @@ projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; p
 cd "${projectRoot}"
 
 docker-compose down -v
+docker-compose -f docker-compose.migrate.yaml down
+docker-compose -f docker-compose.migrate.yaml run database-up
```

Comment:
Would add `--rm` argument to this so container is cleaned up when finished.  

ie 

`docker-compose -f docker-compose.migrate.yaml run --rm database-up`

Reply:
agreed , added now.